### ISSUE TITLE
508-defect-4 [SCREENREADER]: There should be a space after punctuation

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#28354]
# [508-defect-4](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-4)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** Josh

## Details

Without a space after punctuation in "school.They" , the screen reader will awkwardly read the content. 

## Acceptance Criteria

- [ ] Space is added after the period

## Steps to Recreate

1. Log in as user 1 and go to `https://staging.va.gov/manage-va-debt/your-debt/`
2. Expand the expandable alert and confirm there is no space after the period "school.They"

## Screenshots or Trace Logs

<img width="886" alt="Screen Shot 2021-08-05 at 1 56 41 PM" src="https://user-images.githubusercontent.com/14154792/128398572-61bd9dcd-93f4-4d4d-95f4-1e742a002006.png">
